website/src/documentation/programming-guide.md

```diff
@@ -3121,6 +3121,15 @@ public class MyMetricsDoFn extends DoFn<Integer, Integer> {
   }
 }
 ```  
+### 9.5 Export metrics {#export-metrics}
+Beam metrics can be exported to external sinks. If set up the runner will push metrics to a configured sink at a configured period. 
+The configuration is held in the [MetricsOptions](https://beam.apache.org/releases/javadoc/2.19.0/org/apache/beam/sdk/metrics/MetricsOptions.html) class.
+It contains push period configuration and also sink specific options such as URL. As for now only REST http sink and graphite sinks are supported and only
```

Comment:
> As for now only REST http sink and graphite sinks are supported

I's suggest to drop the first "sink", and leave it like:
"As for now, the only REST HTTP and Graphite sinks are supported"

Reply:
+1